index.html

```diff
@@ -0,0 +1,272 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito:wght@200&family=Roboto:ital,wght@0,100;0,300;0,400;0,700;0,900;1,700&display=swap"
+      rel="stylesheet"/>
+
+    <title>Home</title>
+
+  </head>
+  <body class="font-nunito">
```

Comment:
approve your naming, or add the body style in the css file without making it a class